crates/aptos/src/genesis/mod.rs

```diff
@@ -294,6 +295,7 @@ pub fn fetch_genesis_info(git_options: GitOptions) -> CliTypedResult<GenesisInfo
             employee_vesting_start: layout.employee_vesting_start,
             employee_vesting_period_duration: layout.employee_vesting_period_duration,
             consensus_config: OnChainConsensusConfig::default(),
+            execution_config: OnChainExecutionConfig::default(),
```

Comment:
we probably want to enable the shuffling for new genesis?

Reply:
@zekun000 - I will do that as in a follow up PR - will need to run Forge benchmark again with some potential tuning before enabling this by default. Does it make sense?